packages/mui-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/codemod",
-  "version": "6.0.0-alpha.1",
+  "version": "6.0.0-alpha.3",
```

Comment:
We should keep all packages with the `6.0.0-alpha.x` version in the same alpha version, right? This will make it easier for the release manager.

Reply:
(Keep the package versions of stable public packages the same as the root package.json version)